zerver/models.py

```diff
@@ -457,9 +457,7 @@ def get_signup_notifications_stream(self) -> Optional['Stream']:
 
     @property
     def max_invites(self) -> int:
-        if self._max_invites is None:
-            return settings.INVITES_DEFAULT_REALM_DAILY_MAX
-        return self._max_invites
+        return self._max_invites or settings.INVITES_DEFAULT_REALM_DAILY_MAX
```

Comment:
This change will make `_max_invites=0` impossible, which we probably want to support.